apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/AvatarGroup.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v8/Components/AvatarGroup Upgrade" />
+
+# AvatarGroup Upgrade
+
+Fluent UI v8 provides the `Facepile` control to allow users to display a list of `Personas`. Fluent UI v9 provides an `AvatarGroup` control, but has a different API.
+
+Some key differences between `Facepile` and `AvatarGroup` are:
+
+- v9 breaks the component apart into one main component and two subcomponents:
+  - `AvatarGroup`: the main component that is in charge of the inline items displayed in the `AvatarGroup`.
+  - `AvatarGroupPopover`: subcomponent in charge of rendering the overflow indicator button and rendering the `Popover` containing the overflowing `AvatarGroupItems`.
+  - `AvatarGroupItem`: used instead of `Personas` and provides internal functionality for `AvatarGroup`.
+
+v9 also provides a function `partitionAvatarGroupItems` that will split the items provided with the right behavior based on the layout provided. It is highly
+encouraged to use this function as each layout has a specific way of arranging the items.
+
+## Examples
+
+### Basic Upgrade
+
+Basic usage of `FacePile` in v8
+
+```tsx
+import React from 'react';
+import { Facepile, IFacepilePersona } from '@fluentui/react/lib/Facepile';
+import { PersonaSize } from '@fluentui/react/lib/Persona';
+
+const AvatarGroupV8BasicExample = () => {
+  const personas: IFacepilePersona[] = React.useMemo(() => facepilePersonas.slice(0, numberOfFaces), [numberOfFaces]);
```

Comment:
I know we try to keep these examples short but I think they should also be working (i.e., copy-pasteable). `numberOfFaces` and `facepilePersonas` are not defined and should be imo.